internal/utils_test.go

```diff
@@ -106,6 +109,67 @@ data:
 		}
 		assertEqual(t, kind2, "ConfigMap")
 	}
+
+	// test NonConsolidated = true case
```

Comment:
TestGetPolicyTemplate is too long and has too many duplicated codes, let me know if we need to subtract a helper and short it here.

Reply:
It's a nice idea, but I don't want to block this PR on that change.

Feel free to work on this in a separate PR.